docker/linea-mainnet/erigon-genesis.json

```diff
@@ -0,0 +1,806 @@
+{
```

Comment:
I think this should be removed, you seem to use the Geth genesis.json file, so I would remove it.

Reply:
removed